advanced-usage/index.md

```diff
@@ -54,11 +54,50 @@ The current time and date is displayed in the top right corner.
 
 ## Edit Mode
 
-Cockpit's interface is designed around a configurable widget system, within an arbitrary set of views.
-Widgets can be added/removed, placed in arbitrary locations, and resized.
+Cockpit's interface consists of a configurable widget system, with
+1. [Profiles](#profiles)
+    - for supporting different operators and/or vehicle types
+    - can be added/removed, saved and loaded, and switched between in edit mode
```

Comment:
Can also be sent to your vehicle or imported from there, so you don't need to create a new profile for every device that you use.

They can also be exported/imported as a file to your computer, so you can share with other users or other vehicles.

Reply:
Importing is mentioned lower down. Vehicle saving is out of scope for this PR because it's not yet in a release.